ext/templates-settings.html

```diff
@@ -62,6 +62,8 @@
         </button>
     </div>
     <input type="number" step="1" class="short-height dictionary-priority">
+    <button type="button" class="icon-button" id="dictionary-move-up" data-menu-action="moveUp"><span class="icon-button-inner"><span class="icon" data-icon="up-chevron"></span></span></button>
```

Comment:
Is there alt text that we can add to help confused users?

Reply:
If users are getting concerned about what up and down arrows do I'm concerned about them.

Due to the size of the arrows we cant really add a "down" label. Could do "Up" and "Dn" maybe.